config/environment.js

```diff
@@ -117,11 +117,26 @@ module.exports = function(environment) {
         metricsAdapters: [
             {
                 name: 'GoogleAnalytics',
-                environments: ['all'],
+                environments: ['production'],
                 config: {
                     id: process.env.GOOGLE_ANALYTICS_ID
                 }
-            }
+            },
+            {
+                name: 'Keen',
+                environments: ['production'],
+                config: {
+                    private: {
+                        projectId: 'process.env.PREPRINTS_PRIVATE_PROJECT_ID',
```

Comment:
These shouldn't be wrapped in quotes, because they'll get evaluated as literal strings.